scripts/xspdb/pdb-run.py

```diff
@@ -0,0 +1,34 @@
+#***************************************************************************************
+# Copyright (c) 2025 Beijing Institute of Open Source Chip (BOSC)
+# Copyright (c) 2025 Institute of Computing Technology, Chinese Academy of Sciences
+#
+# XiangShan is licensed under Mulan PSL v2.
+# You can use this software according to the terms and conditions of the Mulan PSL v2.
+# You may obtain a copy of Mulan PSL v2 at:
+#          http://license.coscl.org.cn/MulanPSL2
+#
+# THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+# EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+# MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+#
+# See the Mulan PSL v2 for more details.
+#***************************************************************************************
+
+#coding=utf-8
+
+from pyxscore import DUTSimTop, xsp
+from pydifftest import difftest as df
+from xspdb import *
+
+def run_sim_top():
+    dut = DUTSimTop()
+    XSPdb(dut, df, xsp).set_trace()
```

Comment:
a little confused here: it seems `DUTSimTop`, `xsp`, and `df` were all generated by `picker flow`, but we still need to import them, and pass-as-parameter one-by-one.

Can we pack these 3 objects into a single one (e.g. DUTSimTop?), and use `XSPdb(dut).set_trace()` to bootstrap XSPdb?